ports/unixodbc/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "unixodbc",
-  "version-string": "2.3.7",
-  "port-version": 4,
+  "version-string": "2.3.9",
```

```suggestion
  "version": "2.3.9",
```
This might be served by `version`.

---

fixed